lib/app/features/user/model/follow_list.dart

```diff
@@ -52,18 +52,24 @@ class FollowListData with _$FollowListData implements EventSerializable {
 
   factory FollowListData.fromEventMessage(EventMessage eventMessage) {
     return FollowListData(
-      list: eventMessage.tags.map(Followee.fromTag).toList(),
+      list: [
+        for (final tag in eventMessage.tags)
+          if (tag[0] == Followee.tagName) Followee.fromTag(tag),
+      ],
```

Comment:
The same here:

```
list: eventMessage.tags
            .where((tag) => tag[0] == Followee.tagName)
            .map(Followee.fromTag)
            .toList()
```
It's up to you to decide, this is just a suggestion, not a change request.